### PR TITLE
[FIX]  Test case  for Actual Total time  computation 

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

File: project_event/models/project_task.py [hunk 1]
```diff
@@ -224,6 +224,7 @@ class Task(models.Model):
         string='Actual End Time',
     )
     actual_total_time = fields.Char(
+
```
Comment (eilst) [inline project_event/models/project_task.py]: Remove empty line

File: project_event/views/project_task_view.xml [hunk 2]
```diff
@@ -1,15 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="project_event_task_tree_technical_sheet" model="ir.ui.view">
```
Comment (eilst) [inline project_event/views/project_task_view.xml]: You should pull the latest changes, this was removed and you are adding it back. Please Pull the latest 11.0_develop and use it as base

File: project_event/models/project_task.py [hunk 3]
```diff
@@ -224,6 +224,7 @@ class Task(models.Model):
         string='Actual End Time',
```
Comment (eilst) [inline project_event/models/project_task.py]: Please dont do "git add . " You should always chose only the files that you modify, not all.

File: project_event/models/project_task.py (hunk 1 above)
Comment (jananjoy) [reply project_event/models/project_task.py]: @eilst Changed

File: project_event/views/project_task_view.xml (hunk 2 above)
Comment (jananjoy) [reply project_event/views/project_task_view.xml]: @eilst Ok

File: project_event/models/project_task.py (hunk 3 above)
Comment (jananjoy) [reply project_event/models/project_task.py]: @eilst  Ok

**Comment by eilst** [comment]
LVGTM

**Pull request merged**